demo/full/scripts/controllers/Player.jsx

```diff
@@ -105,7 +118,12 @@ class Player extends React.Component {
 
     return (
       <section className="video-player-section">
-        <div className="video-player-content">
+        <div
+          className="video-player-content"
+          style={
+            { width: this.state.videoWidth + "px" }
```

Comment:
Why is this needed at two places at once?

Why is it not needed to set the height?